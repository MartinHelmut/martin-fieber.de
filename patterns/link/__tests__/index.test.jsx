import "jest-styled-components";
import React from "react";
import renderer from "react-test-renderer";

import Link from "../index";

describe("Image", () => {
  test("should render", () => {
    const tree = renderer
      .create(
        <Link href="#" title="Link title">
          Click me
        </Link>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
