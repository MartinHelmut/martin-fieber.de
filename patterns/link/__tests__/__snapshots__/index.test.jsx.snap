// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Image should render 1`] = `
<a
  className="link__Link-sc-1yuyujg-0 bfXUFJ"
  href="#"
  title="Link title"
>
  Click me
</a>
`;
